MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,140 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
+    {
+        // Loop through all the widgets associated with the text box field
+        foreach (var widget in textBoxField.Widgets)
+        {
+            // Set the foreground color of each widget to Aquamarine
+            widget. ForegroundColor = Colors.Aquamarine;
+        }
+    }
+} 
+{% endhighlight %}
+{% endtabs %}
+
+
+
+### Customize the color to the border of the form fields
+
+The `BorderColor` property sets the color of the border around  the form field. For users with color blindness, using high-contrast border colors can improve the visibility of the form fields. Additionally, changing the border color to red for an error state can be more noticeable for those with difficulty distinguishing certain colors. The following code explains how to enable the border color for text form fields. Similarly, you can do the same for other form fields like checkboxes, radio buttons, and other form fields by modifying their respective properties.The following example illustrates how to add a  border color for a text form field by accessing the text form field’s `BorderColor` property. 
+ 
+
+{% tabs %}
+{% highlight C# %} 
+foreach (FormField formField in PdfViewer.FormFields)
+{
+     // Check if the current form field is a text form field 
+    if (formField is TextFormField textBoxField)
+    {
+         // Loop through all the widgets associated with the text box field
+        foreach (var widget in textBoxField.Widgets)
+        {
+            // Set the border color of each widget to blue
+            widget. BorderColor = Colors.Blue;
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Customize the width to the border of the form fields
+
+The `BorderWidth` property sets the thickness of the border around a form field.  Adjusting the border width of form fields can enhance accessibility by making it easier for users with visual impairments to identify and focus on fields. Thicker borders can create a more pronounced visual hierarchy, making the form easier to navigate. The following code explains how to enable the border width for text form fields. Similarly, you can do the same for other form fields like checkboxes, radio buttons, and other form fields by modifying their respective properties.The following example illustrates how to add a border width for a text form field by accessing the text form field’s `BorderWidth` property. 
+ 
+
+{% tabs %}
+{% highlight C# %}
+foreach (FormField formField in PdfViewer.FormFields)
+{
+     // Check if the current form field is a text form field 
+    if (formField is TextFormField textBoxField)
+    {
+        // Loop through all the widgets associated with the text box field
+        foreach (var widget in textBoxField.Widgets)
+        {
+            // Set the border color of each widget to 2.0f
+            widget. BorderWidth = 2.0f;
+        }
+    }
+} 
+{% endhighlight %}
+{% endtabs %}
+
+### PropertyChanged Event for Form field
```

Comment:
Still in the general context, API terminology is used.
<img width="326" alt="image" src="https://github.com/user-attachments/assets/4c78ac4f-b9f1-40ab-9cd4-53149c9ca2b4" />